docs/src/docs/browsercaps/common-browser-caps.adoc

```diff
@@ -23,19 +23,27 @@ WEB_DRIVER_MANAGER.setGlobalCapability(FirefoxOptions.FIREFOX_OPTIONS, options);
 
 ====
 
+= Request capabilities
 
-== Request capabilities
-
-Some WebDriverRequests support setting capabilities, like the `DesktopWebDriverRequest`.
+A request defined capability means using a Webdriver request like `DesktopWebDriverRequest`.
 
+.Set capabilities to a DesktopWebDriverRequest object
 [source,java]
 ----
-WebDriverRequest myRequest = new DesktopWebDriverRequest();
+DesktopWebDriverRequest request = new DesktopWebDriverRequest();
+DesiredCapabilities caps = request.getDesiredCapabilities();
 myRequest.getDesiredCapabilities().put(CapabilityType.ACCEPT_INSECURE_CERTS, true);
-WebDriver webDriver = WEB_DRIVER_MANAGER.getWebDriver(myRequest);
+// Start your session with the DesktopWebDriverRequest object
+WebDriver driver = WebDriverManager.getWebDriver(request);
 ----
 
-== User agent configuration
+[NOTE]
+=====
+Have a look into <<Browser specific knowledge>> for specific browser options. +
+Find some more details for `DesktopWebDriverRequest` at <<_request_configuration>>.
```

Comment:
Anchor `_request_configuration` does not exist? (And also did not seem to exist previously, so unsure whence it should point.)

Reply:
This string is another format for `Request configuration` and it points to `webdrivermanager-config.adoc`

Reply:
GitHub turns this into a relative anchor on this file though: The generated link is `https://github.com/telekom/testerra/blob/docs/add-links-to-tt-version-docs/docs/src/docs/browsercaps/common-browser-caps.adoc#_request_configuration`.